### PR TITLE
Added Jolen

### PR DESCRIPTION
**Pull request by fhamon**

### CODE REVIEW

File: humans.txt [hunk 1]
```diff
@@ -100,6 +100,9 @@ Project manager / Chargée de projet
 Stéphanie Forget
 Content manager - Infographist / Édimestre - Infographiste
 
+Jolen Côté Croteau
+Coordiantor - Coordonnatrice
```
Comment (PascalPiche) [inline humans.txt]: typo.. coordiantor ;)

Comment (PascalPiche) [reply humans.txt]: ping @fhamon

Comment (fhamon) [reply humans.txt]: Good catch

File: humans.txt [hunk 2]
```diff
@@ -100,6 +100,9 @@ Project manager / Chargée de projet
 Stéphanie Forget
 Content manager - Infographist / Édimestre - Infographiste
 
+Jolen Côté Croteau
+Coordinator - Coordonnatrice
```
Comment (nitriques) [inline humans.txt]: We split EN / FR with a forward slash, not a dash.

```suggestion
Coordinator / Coordonnatrice
```

**Pull request merged**